src/pages/not-found/not-found.module.scss

```diff
@@ -0,0 +1,74 @@
+@import '../../styles/variables.module.scss';
+@import '../../styles/fonts.module.scss';
+
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 48px;
+  height: 100%;
+  width: 100%;
+  margin-top: -88px;
+
+  img {
+    width: 100%;
+    max-width: 310px;
+  }
+
+  @media (min-width: $sm) {
+    gap: 64px;
+    margin-top: 0;
+
+    img {
+      max-width: 460px;
```

Comment:
I made this value up so that it looks good. We don't have designs for this screen

Reply:
IDK. I would rather do `max-width: unset` so the tablet looks the same as desktop. It doesn't overflow or break anything.

Alternatively, if you think it's too big for that screen, we can keep this max-width value but then make the font-size also a bit smaller so we are consistent with scaling the page.

Reply:
To me it looks pretty good, not sure why you don't like it.
Let me ask Design Team about it

Reply:
It's ok to scale the graphic down on a tablet screen but why then keep the title and subtitle the same size?

Reply:
But it's a really minor thing. I am ok even with the current thing (that's why I approved the PR)

Reply:
Yeah, it's no big deal. But I already dropped the question so I'll quickly adjust to anything they want. Maybe they'll give us adjusted text sizes too

Reply:
Design Team didn't answer yet. I'll leave it like it is and we can come back to it later if needed